Sources/SpeziOnboarding/OnboardingStack.swift

```diff
@@ -0,0 +1,18 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+// swiftlint:disable file_types_order
```

Comment:
Nit: We should enable this again at the end of the file.

Reply:
does it matter? my understanding is that swiftlint disable directives are at file scope. (will change it, regardless)

Reply:
Nope doesn't matter, but best practice to only disable very selective blocks from swiftlint checking ;)